vividus-plugin-web-app-to-rest-api/src/main/java/org/vividus/steps/integration/ResourceCheckSteps.java

```diff
@@ -157,6 +157,11 @@ private Stream<WebPageResourceValidation> createResourceValidations(Collection<E
     private Optional<WebPageResourceValidation> parseElement(Element element)
     {
         String elementUriAsString = getElementUri(element).trim();
+        if ("img".equals(element.tagName()) && elementUriAsString.startsWith("data:"))
```

Comment:
shouldn't we filtered out all data URLs?

Reply:
as per examples in the integer its used only in images